x-pack/platform/packages/shared/onechat/onechat-server/allow_lists.ts

```diff
@@ -18,6 +18,8 @@ export const AGENT_BUILDER_BUILTIN_TOOLS: string[] = [
   'observability.search_knowledge_base',
   'observability.get_data_sources',
   'observability.get_alerts',
+  'observability.get_services',
+  'observability.get_downstream_dependencies',
```

Comment:
Are those tools "specific" to APM? If so, would it make sense to have a `observability.apm` prefix instead?

Asked on slack, will be better to follow up

Reply:
I asked Viduni to remove "APM" prefix. Responded on slack but the tldr is that we aim to have signal- and collector-agnostic tools.